sdk/Lusid.Sdk.Tests/tutorials/Instruments/InterestRateSwapWithNamedConventionsExamples.cs

```diff
@@ -57,14 +92,14 @@ private void UpsertNamedConventionsToLusid()
         {
             // CREATE the flow conventions and index convention for swap
             string scope = "Conventions";
-            string flowConventionsCode = "GBP-3M";
```

Comment:
FYI: Changing this to USD-6M for convenience. There is no reason to specifically have GBP-3M. We just want an example to work for some swap.